src/main/webapp/WEB-INF/tags/module/purap/notes-sciquest.tag

```diff
@@ -192,7 +192,11 @@
                                             </c:if>
                                             <span>
                                                 <c:if test="${kfunc:canViewNoteAttachment(KualiForm.document, attachmentTypeCode)}">
-                                                    <html:image property="methodToCall.downloadBOAttachment.attachment[${status.index}]" src="${ConfigProperties.kr.externalizable.images.url}clip.gif" title="download attachment" alt="download attachment" style="padding:5px;margin-bottom:-10px;" onclick="excludeSubmitRestriction=true"/>
+                                                	<c:set var="mimeTypeCode" value="${note.attachment.attachmentMimeTypeCode}"/> 

```

Comment:
I think this c:set for the mimeTypeCode should be in the if block above where attachmentTypeCode is being set.